static/bidder-info/vox.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "https://ssp.hybrid.ai/prebid/server/v1/auction"
+maintainer:
+  email: prebid@hybrid.ai
```

Comment:
@Gradi prebid team has sent verification mail on prebid@hybrid.ai . Please respond back with "received" message to complete verification

Reply:
Verification completed!